### DISCUSSION TITLE
Why I have 3 pictures ?

### DISCUSSION THREAD

**Discussion by adriy-be** [LycheeOrg/Lychee#2771] [Q&A] [answered]
I have just done the upgrade to the newest version of lychee.

On the main page, why I have 3 same pictures for each album ?
Is-it a normal behavior ?

If I'm not wrong, it's due to this code : https://github.com/LycheeOrg/Lychee/blob/ba1a43646e5688d0ed80c386da8d84bd11c7b340/resources/js/components/gallery/thumbs/AlbumThumb.vue#L8

![image](https://github.com/user-attachments/assets/bebbe5f8-d6de-4c9c-9ac0-2739556378d8)

Thanks for your help :)

**Comment by ildyria** [comment]
No that is normal, what is not normal is that they are not superposed.

**Comment by adriy-be** [comment, marked as answer]
@ildyria I have found the problem, I have add some custom CSS. Remove my custom CSS fix the problem.
Thanks for your help 😊
